tools/azure-sdk-tools/packaging_tools/generate_utils.py

```diff
@@ -162,12 +162,13 @@ def gen_package_name(origin_config: Dict[str, Any]) -> str:
     return Path(origin_config["output-folder"]).parts[-1]
 
 
-def gen_basic_config(origin_config: Dict[str, Any]) -> Dict[str, Any]:
+def gen_basic_config(origin_config: Dict[str, Any], spec_folder: str = "../../../../../azure-rest-api-specs/") -> Dict[str, Any]:
```

Comment:
put `DEFAULT_SPEC_FOLDER` in util then you could also use it here

Reply:
fixed